### ISSUE TITLE
Authors App is failing to deliver to specific recipients

### ISSUE THREAD

**Issue by reece-r** [OfficeDev/microsoft-teams-apps-company-communicator#1110]
We are experiencing issue with the Author app sending to specific recipients, occurring for multiple instances in the same Teams tenant.

**Observation 1: Blocked by App Permission Policy**
_(Version 5.4.1, all Deployment Center items showing as 'Success' for Web App + 3x Functions)_

When we try to 'Send to all in chat', we have not been able to send without getting this error about Teams App Permission policy. We have confirmed the users that have delivery failed are all in different custom policies for Teams App Permission policy, however the User/Author Teams App is set to ALLOWED in these policies. We have waited over 1-week since adding the User/Author app to the allowed apps in these policies and still have the same error each attempt. We have Proactively Install User App set to true.

![image](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/assets/81006939/c663111f-2226-47b9-bf12-dbc4b28db7ea)

In the screenshot we have an error saying "App is blocked by permission policy" and it lists an AppId. This AppId is not recognised and does not correspond to any ClientId/ObjectId for any Teams Company Communicator apps in our tenant. 

![image](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/assets/81006939/ffa42f8d-2e39-46ed-95dc-ab7cb5d6208e)

**Observation 2: Error in Second Company Communicator instance** 
_(Version 5.3, all Deployment Center items showing as 'Success' for Web App + 3x Functions)_

![image](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/assets/81006939/912ebc72-d8ae-4134-abec-55f20e8f4360)
In our second instance, we are also having issues sending to specific users who have delivery failed. I have attached message delivery logs in the screenshot above.

**Comment by gsv022** [comment]
Hi @reece-r ,
Thanks for raising the query. We will get back to you with proper response soon

**Comment by gsv022** [comment]
@reece-r , Please share your outlook id(email id) so that we can discuss the issue internally.

**Comment by reece-r** [comment]
Hi @gsv022  , please let me know when we can be in contact about this issue, thank you

**Comment by gsv022** [comment]
Hi @reece-r,
We have sent a mail to the given outlook id. We will continue the conversation over the email from now on

**Comment by gsv022** [comment]
Please let us know if you are still facing the issue otherwise we can make it to closure

**Comment by gsv022** [comment]
@reece-r , We are closing this request as we have provided the inputs for the query raised. If you face any issue with company communicator please raise a new issue for further assistance.

**Comment by tomatosis** [comment]
encountered the same issue
